### DISCUSSION TITLE
Install instructions for Ubuntu don't work

### DISCUSSION THREAD

**Discussion by 94131** [45Drives/cockpit-file-sharing#38] [General]
I've been trying to install this module for a couple hours going over the instructions several times. I even tried it from source. 

Getting an error, see below

leo@nas:~$ curl -sSL https://repo.45drives.com/setup | sudo bash
[sudo] password for leo: 
Detected Debian-based distribution. Continuing...
There were no existing 45Drives repos found. Setting up the new repo...
Updating ca-certificates to ensure certificate validity...
Hit:1 http://us.archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:3 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:4 http://us.archive.ubuntu.com/ubuntu jammy-security InRelease
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
ca-certificates is already the newest version (20211016).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Exiting...
leo@nas:~$ sudo apt install cockpit-file-sharing
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package cockpit-file-sharing
leo@nas:~$ 

I'm on Ubuntu server 22.04 and it just craps out

**Comment by marcoczen** [comment]
Hi . Did u every solve this ?
I am on Ubuntu LTS 22.04 too ....

**Reply by 94131** [reply]
nope I gave up because I couldn't find anything for a solution

**Reply by jayfizze** [reply]
I installed on 24.4 using the instructions for Ubuntu 20.04
I did not get the share to accessable so maybe my issue is related to this.

**Comment by OptimusGREEN** [comment]
I dont think ubuntu 22.04 is supported yet, just 20.04

**Comment by bishisht** [comment]
When will 22.04 supported?
